go.mod

```diff
@@ -20,3 +20,78 @@ require (
 	sigs.k8s.io/controller-runtime v0.10.2
 	sigs.k8s.io/yaml v1.2.0
 )
+
+require (
```

Comment:
- any idea why this was factored out into a new `require` block and no changes to `go.sum`?

Reply:
Actually, I ran `go mod tidy` which did this change, and nothing in the go.mod file. As they are all indirect deps we might already have all of them in the go.sum